test/unit/node/util.test.ts

```diff
@@ -0,0 +1,10 @@
+import { getEnvPaths } from "../../../src/node/util"
+
+describe("getEnvPaths", () => {
+  it("should return an object with the data, config and runtime path", () => {
```

Comment:
is it worth testing what the actual results would be for a given homedir? These are only checking that the properties are set, not that they're sensible. On the other hand, it also seems sensible to expect that `xdg-basedir` is well-tested, so maybe it doesn't really matter and these tests are good enough

Reply:
Yes! I think we were reviewing at the same time, but left a comment: https://github.com/cdr/code-server/pull/3304#discussion_r629748398

I agree. We should test the actual paths and simulate different environments.

Reply:
Hmm that's a good point, if we test the actual content it does seem like we're just re-testing `xdg-basedir` to a point which...haha doesn't actually seem to have tests running at the moment: https://github.com/sindresorhus/xdg-basedir/blob/main/test.js